app/src/main/java/com/teamltt/carcare/activity/HomeActivity.java

```diff
@@ -89,7 +95,11 @@ protected void onCreate(Bundle savedInstanceState) {
         // Now start the new service
         startService(btServiceIntent);
 
+        DbHelper helper = new DbHelper(HomeActivity.this);
```

Comment:
a DbHelper instance is created several times in this class. Perhaps it should be a member?